StripeFinancialConnections/StripeFinancialConnections/Source/Native/NativeFlowController.swift

```diff
@@ -392,7 +392,11 @@ extension NativeFlowController: PartnerAuthViewControllerDelegate {
             nativeFlowController: self,
             dataManager: dataManager
         )
-        pushViewController(accountPickerViewController, animated: true)
+        // This is a weird thing to do, but effectively we don't want to
+        // animate for OAuth since we make the authorize call in that case
+        // and already have the same loading screen.
+        let shouldAnimate = !authSession.isOauthNonOptional
```

Comment:
another way to make this more 'functionional' would have been to change the delegate call to pass more info

ex. change it to:

```
 func partnerAuthViewController(
        _ viewController: PartnerAuthViewController,
        didCompleteWithAuthSession authSession: FinancialConnectionsAuthSession
         animateTransition: Bool
    )
```

this makes it a little more robust if logic changes (ex. ensures that this 'animation decision' is directly tied to the code in PartnerAuth pane)

Reply:
I did exactly that at first, but then it seemed weird as this delegate method shouldn't technically know anything about transitions. We are sort of leaking the abstraction within this function implementation to it's caller.